.travis.yml

```diff
@@ -7,7 +7,6 @@ jdk:
 # Run Docker Pulls in parallel, esp for vitess/base with size > 1GB
 install:
   - docker pull squareup/misk-web &
-  - docker pull vitess/base &
```

Comment:
I'd recommend leaving this in so it can pull in the background and keep minimal CI build times as opposed to synchronously waiting for it to pull >1GB image down when it hits the `misk-hibernate` code.